website/integrations/services/freshrss/index.md

```diff
@@ -0,0 +1,63 @@
+---
+title: FreshRSS
+---
+
+<span class="badge badge--secondary">Support level: Community</span>
+
+## What is FreshRSS
+
+> FreshRSS is a self-hosted RSS feed aggregator.
+>
+> -- https://github.com/FreshRSS/FreshRSS
+
+## Preparation
+
+The following placeholders will be used:
+
+-   `service.company` is the FQDN of the FreshRSS install.
```

Comment:
```suggestion
-   `freshrss.company` is the FQDN of the FreshRSS install.
```
And the same for the rest of the document.

Reply:
Done, thanks!